src/pages/TokenDetails/index.tsx

```diff
@@ -1,150 +1,55 @@
-import { Currency, Token } from '@uniswap/sdk-core'
-import { PageName } from 'analytics/constants'
-import { Trace } from 'analytics/Trace'
-import { AboutSection } from 'components/Tokens/TokenDetails/About'
-import AddressSection from 'components/Tokens/TokenDetails/AddressSection'
-import BalanceSummary from 'components/Tokens/TokenDetails/BalanceSummary'
-import { BreadcrumbNavLink } from 'components/Tokens/TokenDetails/BreadcrumbNavLink'
-import ChartSection from 'components/Tokens/TokenDetails/ChartSection'
-import MobileBalanceSummaryFooter from 'components/Tokens/TokenDetails/MobileBalanceSummaryFooter'
-import TokenDetailsSkeleton, {
-  Hr,
-  LeftPanel,
-  RightPanel,
-  TokenDetailsLayout,
-} from 'components/Tokens/TokenDetails/Skeleton'
-import StatsSection from 'components/Tokens/TokenDetails/StatsSection'
-import TokenSafetyMessage from 'components/TokenSafety/TokenSafetyMessage'
-import TokenSafetyModal from 'components/TokenSafety/TokenSafetyModal'
-import Widget from 'components/Widget'
-import { DEFAULT_ERC20_DECIMALS, NATIVE_CHAIN_ID, nativeOnChain } from 'constants/tokens'
-import { checkWarning } from 'constants/tokenSafety'
-import { Chain } from 'graphql/data/__generated__/TokenQuery.graphql'
-import { QueryToken, useTokenQuery } from 'graphql/data/Token'
-import { useTokenPriceQuery } from 'graphql/data/TokenPrice'
-import { CHAIN_NAME_TO_CHAIN_ID, validateUrlChainParam } from 'graphql/data/util'
-import { useIsUserAddedTokenOnChain } from 'hooks/Tokens'
-import { useOnGlobalChainSwitch } from 'hooks/useGlobalChainSwitch'
-import { useCallback, useMemo, useState, useTransition } from 'react'
-import { ArrowLeft } from 'react-feather'
-import { useNavigate, useParams } from 'react-router-dom'
+import { filterTimeAtom } from 'components/Tokens/state'
+import TokenDetails from 'components/Tokens/TokenDetails'
+import { TokenDetailsPageSkeleton } from 'components/Tokens/TokenDetails/Skeleton'
+import { NATIVE_CHAIN_ID, nativeOnChain } from 'constants/tokens'
+import { TokenPriceQuery } from 'graphql/data/__generated__/TokenPriceQuery.graphql'
```

Comment:
TokenPriceQuery and TokenQuery should be re-exported through `graphql/data/{Token,TokenPrice}` to avoid having to find them in a generated folder.

You _could_ alias `useQueryLoader<TokenQuery>(tokenQuery)` to `useTokenQueryLoader` (similarly with `usePreloadedTokenQuery`), but that's up to you.